src/components/containers/TraceAccordion.js

```diff
@@ -24,14 +24,36 @@ class TraceAccordion extends Component {
     // we don't want to include analysis transforms when we're in the create panel
     const base = props.canGroup ? context.fullData : context.data;
 
-    this.filteredTracesFullDataPositions = [];
-    this.filteredTraces = base.filter((t, i) => {
-      if (props.excludeFits) {
-        return !(t.transforms && t.transforms.every(tr => tr.type === 'fit'));
+    this.filteredTracesFullDataPositions = base.map((t, i) => {
+      if (
```

Comment:
I don't agree with this logic.
When base is data, like in the Create panel, you cannot assume that the index of an item in the data array is the same as its index in the fullData array.

The case I can think of is in the Create Panel, which has base data,
if you add 1 split trace, and 1 non split case.

Data will be like: [{type: 'scatter'}, {type: 'pie'}]
FullData will be like: [{type: 'scatter', transforms: []}, {type: 'scatter', transforms: []}, {type: 'pie'} ]

In the Create panel, you'd be mapping on data, so you'd have a this.filteredTracesFulDataPositions of [0, 1], where you should have [0, 2].